src/Templates/v1/ModelsTemplate.cshtml

```diff
@@ -23,10 +49,10 @@ package @Model.Namespace
 
 @EmptyLine
 
-@if (!imports.IsNullOrEmpty())
+@if (!Model.ModelImports.IsNullOrEmpty())
 {
 @:import (
-foreach (var import in imports)
+foreach (var import in Model.ModelImports)
```

Comment:
I'm a little confused looking at this :/ Admittedly, I could figure it out if I augmented my review here in the browser with a little time pulling down more Autorest bits, but it's easy enough to just ask here!

Is `Model.ModelImports` a string or an Enumerable<string>? Both referencing `.IsNullIOrEmptry() ` and individual imports is throwing me off.

Also, who bother checking to see if there are `Any()`? Wouldn't the loop just not execute? The answer is probably tied in to the first question, but regardless, I don't see the answer :)

Reply:
It's an IEnumerable<string>.  Presumably it could be null which is why the null check but I didn't want to dig into it.
I should also state that while I do this refactoring I don't really intend to fix crappy code like this as sometimes such fixes inadvertently introduce breaking changes.  I'm trying to preserve the original code as much as possible so that things continue to work the way they do today.

Reply:
Got it, that totally makes sense. There's probably an extension method called `IsNullOrEmpty()` that we've defined. I'll keep in mind that we're trying not to touch stuff.